scripts/snfoundryup

```diff
@@ -32,6 +32,26 @@ Options:
   -c, --commit           Specify a specific commit hash to install from
 
 For more information, check out https://foundry-rs.github.io/starknet-foundry/getting-started/installation.html
+EOF
+}
+finished_installation() {
```

Comment:
this is just printing the info, let's name it after that

Reply:
renamed to `print_docs_and_community_info`